src/main/java/com/zhanganzhi/chathub/adaptors/velocity/VelocityAdaptor.java

```diff
@@ -0,0 +1,157 @@
+package com.zhanganzhi.chathub.adaptors.velocity;
+
+import java.util.Arrays;
+
+import com.google.common.eventbus.Subscribe;
```

Comment:
为什么是google的Subscribe？

```suggestion
import com.velocitypowered.api.event.Subscribe;
```

Reply:
我说怎么接收不到velocity的事件，原来这里错了。。